python/ray/serve/tests/test_standalone.py

```diff
@@ -550,12 +550,14 @@ def hello(_):
     def world(_):
         return "world"
 
-    def check(name):
+    def check(name, raise_error=False):
```

Comment:
why do we need the flag? why not just always raise? don't see it used anywhere else

Reply:
`check()` is used in `wait_for_condition()` call near the end of the function. If it always raises, `wait_for_condition()` will fail if `check()` fails on the first try.